src/bvals/bvals_refine.cpp

```diff
@@ -140,10 +113,9 @@ void BoundaryValues::ProlongateGhostCells_(const NeighborBlock &nb, int si, int
 
   for (auto cc_var : pmr->pvars_cc_) {
     if (!cc_var->IsAllocated()) continue;
-    ParArrayND<Real> var_cc = cc_var->data;
-    ParArrayND<Real> coarse_cc = cc_var->coarse_s;
-    int nu = var_cc.GetDim(4) - 1;
-    pmr->ProlongateCellCenteredValues(coarse_cc, var_cc, 0, nu, si, ei, sj, ej, sk, ek);
+    int nu = cc_var->GetDim(4) - 1;
+    pmr->ProlongateCellCenteredValues(cc_var.get(), cc_var.get(), 0, nu, si, ei, sj, ej,
+                                      sk, ek);
```

Comment:
Yeah I think we can pass it in only once. I considered leaving it split because I was thinking maybe we'd want coarse and fine in different stages or something. But for now that's probably overkill.